test/UserService.Validation.UnitTests/Credentials/CreateCredentialsRequestValidatorTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using FluentValidation.TestHelper;
+using LT.DigitalOffice.UserService.Models.Dto.Requests.Credentials;
+using LT.DigitalOffice.UserService.Validation.Credentials;
+using NUnit.Framework;
+using System;
+
+namespace LT.DigitalOffice.UserService.Validation.UnitTests.Credentials
+{
+    class CreateCredentialsRequestValidatorTests
+    {
+        private CreateCredentialsRequestValidator _validator;
+        private CreateCredentialsRequest _request;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _validator = new CreateCredentialsRequestValidator();
+
+            _request = new CreateCredentialsRequest
+            {
+                Login = "Login1234567890",
+                Password = "Password",
+                UserId = Guid.NewGuid()
+            };
+        }
+
+        [Test]
+        public void ShouldNotThrowAnyException()
+        {
+            _validator.TestValidate(_request).ShouldNotHaveAnyValidationErrors();
+        }
+
+        [Test]
+        public void ShouldThrowExceptionWhenAllIsNotCorrect()
+        {
+            _request = new CreateCredentialsRequest
+            {
+                Login = string.Empty,
+                Password = string.Empty,
+                UserId = Guid.Empty
+            };
+
+            _validator.TestValidate(_request).ShouldHaveValidationErrorFor(x => x.Login);
+            _validator.TestValidate(_request).ShouldHaveValidationErrorFor(x => x.Password);
+            _validator.TestValidate(_request).ShouldHaveValidationErrorFor(x => x.UserId);
+        }
+
+        [Test]
```

Comment:
https://docs.microsoft.com/ru-ru/dotnet/core/testing/unit-testing-with-nunit
read this, there are example how write less tests with same logic, I talk about TestCase